### PR TITLE
Update Jest Config and clean up spec files

### PR DESCRIPTION
**Pull request by cschucode**
* Add runtime generator to jest config (should be able to use **async/await** in test files now
* Rename Jest Example Test (will probably be removed anyway) 🤷‍♂️

### CODE REVIEW

File: client/tests/RelatedProducts.test.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import RelatedProducts from '../components/relatedProducts/RelatedProducts';
+
+describe('Related Products', () => {
+  const mockProductId = 43432;
+  const mockRatiing = 3.5;
```
Comment (jleiandy) [inline client/tests/RelatedProducts.test.jsx]: did you mean "mockRating"?

File: client/tests/RelatedProducts.test.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  beforeEach(() => {
+    render(<RelatedProducts productId={mockProductId} rating={mockRatiing} />);
```
Comment (jleiandy) [inline client/tests/RelatedProducts.test.jsx]: interestingly you used the same typo here so this probably worked

**Review by jleiandy — CHANGES_REQUESTED** [review]
Overall looks good, found a small typo you might want to fix first.

File: client/tests/RelatedProducts.test.jsx (hunk 1 above)
Comment (cschucode) [reply client/tests/RelatedProducts.test.jsx]: That's not how you spell it? 🤔

File: client/tests/RelatedProducts.test.jsx (hunk 2 above)
Comment (cschucode) [reply client/tests/RelatedProducts.test.jsx]: At least I'm a consistent bad speller. 😜

**Review by jleiandy — APPROVED** [review]

**Pull request merged**